x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/RetrySearchIntegTests.java

```diff
@@ -87,10 +87,8 @@ public void testSearcherId() throws Exception {
             }
         }
 
-        for (String allocatedNode : allocatedNodes) {
-            if (randomBoolean()) {
-                internalCluster().restartNode(allocatedNode);
-            }
+        if (randomBoolean()) {
+            internalCluster().restartNode(randomFrom(allocatedNodes));
         }
```

Comment:
Can we retain the possibility for restarting all nodes by adding ensureGreen after each restart?

Reply:
Yeah, that was one options that I considered. I'm not sure whether it matters for the test if restart one or multiple nodes, since we just check that the search id and PIT are retained during node restarts, but I will update it to keep the original semantics of the test

Reply:
ad8369bbd48e818e3d782e81170122b381efa9e2